Sprintfinity3902/PauseMenu.cs

```diff
@@ -163,6 +164,14 @@ public void ChangePosition()
                 }
             }
 
+            foreach (IEntity door in Game.dungeon.GetCurrentRoom().doors)
+            {
+                if (count != 176 * Global.Var.SCALE && Pause)
+                    door.Y = door.Y + 2 * Global.Var.SCALE;
+                else if (count != 176 * Global.Var.SCALE && Pause == false)
+                    door.Y = door.Y - 2 * Global.Var.SCALE;
```

Comment:
Magic numbers need to be fixed here.

Reply:
Looks like the entire PauseMenu.cs still needs a magic numbers rework so I'm fine with merging this.